doc/content/examples/example11b.md

```diff
@@ -0,0 +1,94 @@
+
+!media media/examples/ex11b/ex11b_indicatormarker.png
+      style=width:50%;margin-left:150px;float:center;
+      id=fig:beam_indicatormarker
+      caption=Beam model with the green block's mesh refined using the [`MinimumElementSizeMarker`](markers/MinimumElementSizeMarker.md) with the `indicator` option.
+
+### Mesh refinement using the `MinimumElementSizeMarker` with the `element_size` option
+
+Alternatively, instead of relying on an indicator, the [`MinimumElementSizeMarker`](markers/MinimumElementSizeMarker.md) can directly take an `element_size` value for the mesh refinement. This capability is demonstrated here. The goal is refine the red subdomain in the beam such that the minimum element size is less than or equal to 0.15 m. This can be accomplished using the `element_size` option as shown in the code block below.
+
+!listing examples/ex11b/3d_beam_static_refine_PP.i
+         start=marker1
+         end=marker2
+
+The end result, as presented in the figure below, is the red subdomain having a element size less than 0.15 m. Additionally, it is worth noticing that the green subdomain is also refined near the interface with the red subdomain. However, we did not intend to refine the green subdomain here. This behavior is due to the nature of $h$-adaptivity which MOOSE employs. For more information, please refer to the [Adaptivity System](syntax/Adaptivity/index.md).
+
+!media media/examples/ex11b/ex11b_valuemarker.png
+      style=width:50%;margin-left:150px;float:center;
+      id=fig:beam_valuemarker
+      caption=Beam model with the red block's mesh refined using the [`MinimumElementSizeMarker`](markers/MinimumElementSizeMarker.md) with the `element_size` option.
+
+### Mesh refinement using the `ComboMarker`
+
+It is also possible to combine two different `Markers` into a single `Marker` with the [`ComboMarker`](/markers/ComboMarker.md) option. The goal now is to combine the `MinimumElementSizeMarker` with the `indicator` option and `MinimumElementSizeMarker` with the `element_size` option. The below code block accomplishes this.
+
+!listing examples/ex11b/3d_beam_static_refine_PP.i
+         start=Markers
+         end=GlobalParams
+
+The figure below presents the mesh of the beam after refinement using the [`ComboMarker`](/markers/ComboMarker.md). Notice that even though the goal is to refine the red subdomain to have an element size less than 0.15 m and the green subdomain to have an element size less than 0.25 m, both these subdomains have an element size less than 0.15 m. This behavior is due to how the [`ComboMarker`](/markers/ComboMarker.md) operates. Given several `Markers`, the [`ComboMarker`](/markers/ComboMarker.md) refines all the associated subdomains based on the largest refinement required across all the given `Markers`.
```

Comment:
I think the last statement is a little confusing. Maybe something like "the mesh for each of the associated subdomains is refined to the maximum level of refinement allowed by the limiting marker." If I am getting that correctly...

Reply:
Done!